test/pkg.jl

```diff
@@ -88,6 +88,57 @@ import Pkg.Types: semver_spec, VersionSpec
     @test   v"1.2.3" in semver_spec(">=1.2.3")
     @test !(v"1.2.2" in semver_spec(">=1.2.3"))
 
+    @test semver_spec("0.1.0 - 0.2.2") == VersionSpec("0.1.0 - 0.2.2")
```

Comment:
There are 3x3 branches in the code so the number of tests here feel a bit excessive.

Reply:
Will keep the tests per triage